src/views/ManageList.jsx

```diff
@@ -58,13 +78,28 @@ export function ManageList({ listPath, userId }) {
 		}
 	};
 
+	//Alerts based on "submitted" value
+	const alertText = (submittedValue) => {
+		switch (submittedValue) {
+			case 'added':
+				return <span>Your item was added!</span>;
+			case 'failed':
+				return <span>Your item wasn't added!</span>;
+			case 'empty':
+				return <span>No empty items!</span>;
```

Comment:
We could update the message here - e.g. `Please enter an item to add to your list` or something like that. Also fine to leave this for now and add as a todo to revisit later.

Reply:
Updated! Thank you for suggesting the change. Much more helpful that the one we came up with.